pkg/log/log.go

```diff
@@ -77,6 +77,11 @@ func initializeLogger() error {
 		} else {
 			setFormatter(FormatLayoutText)
 		}
+
+		level := os.Getenv("JX_LOG_LEVEL")
```

Comment:
> BTW, I'd recommend using `os.LookupEnv`

I used `os.Getenv` because that one is also used to read `JX_LOG_FORMAT`.

> SetLevel is called when the --verbose flag is specified which is after the logger is initialised.

Ok, I changed my code to respect the following: if the environment variable is set, then it has the precedence, otherwise the old logic applies.